manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -271,7 +273,7 @@ spec:
               serviceAccountName: kiali-operator
               containers:
               - name: operator
-                image: ${KIALI_OPERATOR_REGISTRY}
```

Comment:
KIALI_OPERATOR_REGISTRY is also used in https://github.com/kiali/kiali-operator/blob/e2b3dcc6c99e89bbf165dae62bf24aff7334737a/Makefile#L85